### PR TITLE
Database save fix

### PR DESCRIPTION
**Pull request by Sindahlof**
Fixed how database saves data

### CODE REVIEW

File: CarboPlanner/src/views/HomeScreen.tsx [hunk 1]
```diff
@@ -20,55 +21,21 @@ export default function HomeScreen(){
...
     navigation.navigate('Camera');
   }
+
+  const [data, setData] = useState('')
+
+  useEffect(() => {
+    const unsubscribe = onSnapshot(mainDocRef, (snapshot) => {
+        if (snapshot.exists()) {
+            const fetchedData = snapshot.data().data;
+            console.log(fetchedData[0])
```
Comment (Nolle10) [inline CarboPlanner/src/views/HomeScreen.tsx]:
```suggestion
```

File: CarboPlanner/src/views/ProfileScreen.tsx [hunk 2]
```diff
@@ -1,11 +1,32 @@
...
 export default function ProfileScreen() {
+    const today = new Date();
+    const date = today.getDate() + "-" + (today.getMonth() + 1) + "-" + today.getFullYear();
+    const mainDocRef = doc(db, process.env.EXPO_PUBLIC_UUID, date);
+
+    useEffect(() => {
+        const unsubscribe = onSnapshot(mainDocRef, (snapshot) => {
+            if (snapshot.exists()) {
+                const fetchedData = snapshot.data().data;
+                console.log("Data changed:", fetchedData[0].meals);
```
Comment (Nolle10) [inline CarboPlanner/src/views/ProfileScreen.tsx]:
```suggestion
```

File: CarboPlanner/src/views/ResultScreen.tsx [hunk 3]
```diff
@@ -59,8 +62,18 @@ export default function Loading({route}) {
...
+                ...item,
+                // Set default values for properties other than confidence and name
+                weight: '100',
+                carbs: '33',
+                protein: '12',
+                fat: '8',
+            }));
+            await setItem(initialItems);
         }
+        console.log(data)
```
Comment (Nolle10) [inline CarboPlanner/src/views/ResultScreen.tsx]:
```suggestion
```

**Review by Nolle10 — CHANGES_REQUESTED** [review]
Some console.logs that should be deleted before begin merged

**Comment by Sindahlof** [comment]
Should be fixed now

**Review by Nolle10 — APPROVED** [review]
looks good

**Pull request merged**